README.md

```diff
@@ -4,3 +4,21 @@ Docker files to build dockers based on:
 * OCB code
 
 This work is original based on the docker from http://www.xcg-consulting.fr and available here: https://hub.docker.com/r/xcgd/odoo/
+
+oca_dependencies.txt
+
```

Comment:
@noahzaozao I agree with @elicoidal , please add more info about:

- how to use the image (what are the different commands?)
- what are the ENV variables in the `docker-compose.yml`, what is their purpose?
- what volumes are available, what is their purpose?

As for the auto-dependencies, please explain how it works. e.g.:

- what structure the main GH repo needs to have?
- explain the recursive repo pulling
- explain that if the repo was already cloned, it will be pulled
- etc.